yt_dlp/downloader/external.py

```diff
@@ -293,11 +294,13 @@ def _call_downloader(self, tmpfilename, info_dict):
         return super()._call_downloader(tmpfilename, info_dict)
 
     def _make_cmd(self, tmpfilename, info_dict):
-        cmd = [self.exe, '-c', '--no-conf',
+        cmd = [self.exe, '--no-conf', '--stderr=true',
+               '--auto-save-interval=10', '--allow-overwrite=true', '--always-resume=false',
+               '--auto-file-renaming=false', '--force-save=false',
```

Comment:
Pls explain why you believe each of these is suitable for us. The PR description does not cover all of them

Reply:
I'll add each of the new flags to the description.